service/prometheus/update_test.go

```diff
@@ -37,79 +37,6 @@ func Test_Prometheus_isManaged(t *testing.T) {
 			isManaged: false,
 		},
 
```

Comment:
The following diff is pretty bad. Essentially, we only support the newer scrapeconfigs when checking for updates.